src/content/tutorials/user-auth-with-auth0/_index.md

```diff
@@ -486,16 +489,16 @@ While a user's authentication cookie expires after a day, you may want to offer
 // workers-site/auth0.js
 
 export const logout = event => {
-  const cookieHeader = event.request.headers.get("Cookie");
+  const cookieHeader = event.request.headers.get('Cookie')
   if (cookieHeader && cookieHeader.includes(cookieKey)) {
     return {
       headers: {
-        "Set-cookie": `${cookieKey}="";`
-      }
-    };
+        'Set-cookie': `${cookieKey}="";`,
```

Comment:
Though it seems silly the same cookie directives should be placed here as above (`HttpOnly; SameSite=Lax; Secure;`) to be consistent and ensure there isn't a vector for extracting cookies.